jwst/wfs_combine/tests/test_wfs_combine.py

```diff
@@ -4,18 +4,36 @@
 from jwst.assign_wcs import AssignWcsStep
 from jwst.associations.asn_from_list import asn_from_list
 from jwst.wfs_combine import WfsCombineStep
+from jwst.wfs_combine import wfs_combine
+import numpy as np
 
 
 GOOD = datamodels.dqflags.pixel["GOOD"]
 DO_NOT_USE = datamodels.dqflags.pixel["DO_NOT_USE"]
 SATURATED = datamodels.dqflags.pixel["SATURATED"]
 
 
+def gaussian(x, mu, sig):
+    return 1. / (np.sqrt(2 * np.pi) * sig) * np.exp(-np.power((x - mu) / sig, 2) / 2)
+
+
+def gaussian2d(x, y, mux, muy, sigx, sigy):
+    return gaussian(x, mux, sigx) * gaussian(y, muy, sigy)
+
+
+def add_point_source(inarray, scale, xcen, ycen, sigx, sigy):
+    outarray = np.zeros_like(inarray)
+    for y in range(inarray.shape[0]):
+        for x in range(inarray.shape[1]):
+            outarray[y, x] = scale * gaussian2d(x, y, xcen, ycen, sigx, sigy)
+    return outarray
+
+
 @pytest.fixture(scope="module")
-def wfs_association(tmp_path_factory):
+def wfs_association(tmp_path_factory, imsize=10):
```

Comment:
pytest fixtures should not get arguments, i.e. `imsize=10` does not belong here.

Reply:
done